eslint.config.js

```diff
@@ -1,7 +1,8 @@
 const eslint = require("@eslint/js");
 const n = require("eslint-plugin-n");
+const tsESLint = require("typescript-eslint");
```

Comment:
[Style] From https://typescript-eslint.io/getting-started, we generally call it this:

```suggestion
const tseslint = require("typescript-eslint");
```

Reply:
Updated